### PR TITLE
Fix tags

### PR DESCRIPTION
**Pull request by pzygielo**

### CODE REVIEW

File: api/src/main/java/jakarta/servlet/ServletRequest.java [hunk 1]
```diff
@@ -359,6 +359,7 @@
      * 
      * @deprecated As of Version 2.1 of the Java Servlet API, use {@link ServletContext#getRealPath} instead.
      */
+    @Deprecated
```
Comment (stuartwdouglas) [inline api/src/main/java/jakarta/servlet/ServletRequest.java]: This will need a change in the TCK signature tests.

Comment (pzygielo) [reply api/src/main/java/jakarta/servlet/ServletRequest.java]: Does it mean that at successor of [javax.servlet.sig_4.0_se8:597](https://github.com/eclipse-ee4j/jakartaee-tck/blob/e2c07be584be7a2b9a5f59cfd9651efd2307d944/src/com/sun/ts/tests/signaturetest/signature-repository/javax.servlet.sig_4.0_se8#L597) something like [javax.servlet.sig_4.0_se8:666](https://github.com/eclipse-ee4j/jakartaee-tck/blob/e2c07be584be7a2b9a5f59cfd9651efd2307d944/src/com/sun/ts/tests/signaturetest/signature-repository/javax.servlet.sig_4.0_se8#L666) is done for https://github.com/eclipse-ee4j/servlet-api/blob/1b41595710232879a0a6da86cc21fcba3693723d/api/src/main/java/jakarta/servlet/ServletRequestWrapper.java#L288-L289 once signatures for jakarta.servlet 5.0 are created?

Comment (markt-asf) [reply api/src/main/java/jakarta/servlet/ServletRequest.java]: We can pick this up when we update the TCK for the javax -> jakarta name change.
This [PR to do the same for EL](https://github.com/eclipse-ee4j/jakartaee-tck/pull/150) should give an idea of what is involved if someone wants to volunteer to do this for Servlet.
We should fix any other binary compatible niggles (e.g. missing generics if any) first.

**Comment by pzygielo** [comment]
Thanks for checking.
Let me re-PR the second commit separately then.
Will re-open for annotation if still needed when TCK is ready.

**Comment by markt-asf** [comment]
Sorry, I expressed myself badly. I meant we could pick up fixing the TCK to take account of this change when we update the TCK for the package rename.I didn't mean we needed to delay this PR until then. I'm in favour of merging this now.

**Comment by pzygielo** [comment]
@markt-asf It was me that misunderstood the message :wink:

**Comment by pzygielo** [comment]
May I ask to move it forward, please? (In any direction.)

**Comment by stuartwdouglas** [comment]
Given that @markt-asf has said he was in favour of merging this now I am going to merge this.

**Pull request merged**